Stripe/STPCard.m

```diff
@@ -135,6 +137,54 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+#pragma mark - Description
+
+- (NSString *)description {
+    NSString *fundingDescription;
+
+    switch (self.funding) {
```

Comment:
Yeah I agree we can improve here. A few questions about the current pattern:

- Should we be friendly and `lowercaseString` or be more strict?
  - I would lean towards being more strict
- Should every enum have an "Unknown" case? We would need to add one in this case for `funding`.
  - Feels like adding "Unknown" makes downstream code more complicated
- More generally, I feel like we can come up with something that looks more elegant. Maybe leverage the fact that tests will always be run?